lib/apiForSheets.js

```diff
@@ -128,6 +155,54 @@ module.exports.generateOutput = async function (rows, prNumber, prDetails) {
   }
 };
 
+module.exports.checkClaAndOnboardingTracker = async function
+(authCla) {
+  var sheets = google.sheets('v4');
+  var responseFromCla, responceFromOnBoardingSheet;
+  await sheets.spreadsheets.values.get(
+    {
+      auth: authCla,
+      spreadsheetId: spreadsheetId,
+      range: 'Usernames!A:A',
+    },
+    async function (err, response) {
+      if (err) {
+        console.log('The API returned an error: ' + err);
+        return;
+      }
+
+      responseFromCla = response;
+    }
+  );
+  await sheets.spreadsheets.values.get(
+    {
+      auth: authCla,
+      spreadsheetId: onBoardingSheetId,
+      range: 'Tracker!B:B',
```

Comment:
What about the part of the onboarding sheet where we list all existing contributors (the ones that are already onboarded).